charts/templates/statefulset.yaml

```diff
@@ -51,18 +51,21 @@ spec:
               exit 0
             fi
             log "Info: [initContainer] Group $MARKLOGIC_GROUP does not exist, configuring group $MARKLOGIC_GROUP on the MarkLogic cluster."
-            for i in $(seq 1 5); do
-              res_code=`curl --anyauth --user ${MARKLOGIC_ADMIN_USERNAME}:${MARKLOGIC_ADMIN_PASSWORD} -m 20 -s -w '%{http_code}' -X POST -d "${GROUP_CFG}" -H "Content-type: application/json" http://${MARKLOGIC_BOOTSTRAP_HOST}:8002/manage/v2/groups`
+              res_code=`curl --retry 5 --retry-all-errors --retry-max-time 60 --anyauth --user ${MARKLOGIC_ADMIN_USERNAME}:${MARKLOGIC_ADMIN_PASSWORD} -m 20 -s -w '%{http_code}' -X POST -d "${GROUP_CFG}" -H "Content-type: application/json" http://${MARKLOGIC_BOOTSTRAP_HOST}:8002/manage/v2/groups`
               if [[ ${res_code} -eq 201 ]]; then
                 log "Info: [initContainer] Successfully configured group $MARKLOGIC_GROUP on the MarkLogic cluster."
-                break
               else
-                log "Info: [initContainer] Configure group $MARKLOGIC_GROUP retry attempt $i "
-                log "Info: [initContainer] Expected response code 202, got $res_code"
-                sleep 10s
+                log "Info: [initContainer] Expected response code 201, got $res_code"
+                exit 1
               fi
-              [ $i -eq 5 ] && exit 1
-            done
+            log "Info: [initContainer] Group $MARKLOGIC_GROUP has been created, configuring App-server App-Services in group $MARKLOGIC_GROUP on the MarkLogic cluster."
+              res_code=`curl --retry 10 --retry-all-errors --retry-max-time 60 --anyauth --user ${MARKLOGIC_ADMIN_USERNAME}:${MARKLOGIC_ADMIN_PASSWORD} -m 20 -s -w '%{http_code}' -X POST -d '{"server-name":"App-Services", "root":"/", "port":8000,"modules-database":"Modules", "content-database":"Documents", "error-handler":"/MarkLogic/rest-api/8000-error-handler.xqy", "url-rewriter":"/MarkLogic/rest-api/8000-rewriter.xml"}' -H "Content-type: application/json" "http://${MARKLOGIC_BOOTSTRAP_HOST}:8002/manage/v2/servers?group-id=${MARKLOGIC_GROUP}&server-type=http"`
```

Comment:
This block shouldn't be indented

Reply:
I didn't change the indentation.
It is like this already in the develop branch.

Reply:
Since you removed the outer for loop, you need to move the indentation back for the lines that are in those blocks. Align with the "log" statements on 53 and 61.